lib/checks/aria/valid-scrollable-semantics.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "valid-scrollable-semantics",
+  "evaluate": "valid-scrollable-semantics.js",
+  "options": [],
+  "metadata": {
+    "impact": "minor",
+    "messages": {
```

Comment:
I think it might be confusing to make this check about scrollable regions. The element it pulls up may not be scrollable at all. I'm thinking we do something like "Element does not have a landmark role appropriate for a focusable element". What do you think?

Reply:
I agree the existing message isn't great, but saying "does not have a landmark role..." doesn't capture the tag names also being OK, and we would probably want to say "in the focus order" or "in the tab order" instead of "focusable" -- since "focusable" elements with tabindex=-1 aren't a concern.

Happy to give you final say on the message, leaving for now but let me know if you have a replacement in mind

Reply:
I think you're both onto something here. I vote for a compromise that refers to the element's focusability instead of scrollability (I think this will be more easily apprehended by future readers), but does not mention role to the exclusion of tagname. How about, "Element has valid semantics for a focusable element"? 

This suggestion extends to the names of the map constants (`VALID_{TAG_NAMES|ROLES}_FOR_SCROLLABLE_REGIONS`) and the name of the check itself. Using language that refers to focus instead of scrolling will resonate more with the accessibility community, in my opinion, and will make things easier for everyone to follow.

Reply:
@0ddfell0w You make a good point about the term "focusable". I don't think it's a given that readers would fail to see the implicit distinction between _naturally_ and _programmatically_ focusable elements, but they might.

Maybe "Element has valid semantics for an element in the focus order"?

Reply:
I agree. Let's go with that. I think that's the last change we need for this PR.

Reply:
@WilcoFiers @isner done